voting/src/voting.rs

```diff
@@ -46,6 +76,38 @@ impl Context {
     fn meeting_mut(&mut self) -> &mut dyn GeneralMeetingManager {
         self.general_meeting.as_mut().unwrap().as_mut()
     }
+
+    fn excute_tx(&mut self, transaction: &Transaction) -> Result<H256, ExecuteError> {
+        match transaction.tx_type() {
+            VOTE_PAPER_TX_TYPE => {
+                let tx: PaperOwnTransaction =
+                    serde_cbor::from_slice(&transaction.body()).map_err(|_| ExecuteError::InvalidFormat)?;
+                tx.verify().map_err(|_| ExecuteError::InvalidSignature)?;
+
+                let meeting_id = tx.tx.action.general_meeting_id;
+                let name = tx.tx.action.voter_name;
+                let shares = tx.tx.action.number_of_shares;
+                let agendas = tx.tx.action.agenda_number;
+                let voter_pub_key = tx.tx.action.voter_public_key;
+
+                let meeting: GeneralMeeting =
+                    self.meeting_mut().get_meeting(&meeting_id).map_err(|_| ExecuteError::InvalidMeeting)?;
+                if meeting.get_tallying_time().get_time() < self.block_header.as_ref().unwrap().timestamp() {
+                    return Err(ExecuteError::VoteAfterTallyingTime)
+                }
+                if 0 == agendas && agendas > meeting.get_agendas() {
+                    return Err(ExecuteError::InvalidNumberofAgendas)
```

Comment:
When I read the `InvalidNumberofAgendas` error name, I understood that the number of agendas in a general meeting is wrong. However, it isn't.
agenda_number in the TxCreateVotePaper is wrong. How about `InvalidAgendaNumber`?